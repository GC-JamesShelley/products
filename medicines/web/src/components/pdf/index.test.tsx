import { shallow } from 'enzyme';
import React from 'react';
import Pdf from './index';

describe(Pdf, () => {
  it('should render', () => {
    const component = shallow(<Pdf />);
    expect(component).toMatchSnapshot();
  });
});
