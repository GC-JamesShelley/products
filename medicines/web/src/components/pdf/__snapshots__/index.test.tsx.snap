// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pdf should render 1`] = `ShallowWrapper {}`;
